### PR TITLE
:sparkles: Add Message Preview component

### PR DESCRIPTION
**Pull request by infoucarsinan**
Add Message Preview

### CODE REVIEW

File: src/app/components/MessagePreview/MessagePreview.module.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (phlgr) [inline src/app/components/MessagePreview/MessagePreview.module.css:4]: This not only affects this file but all as this is a universal selector.
Try selecting everything in the container instead

File: src/app/components/MessagePreview/MessagePreview.module.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  grid-row: 1/3;
+}
+
+.userMessage {
+  grid-row: 2;
+}
+
+.userName {
+  grid-row: 1;
+}
```
Comment (phlgr) [inline src/app/components/MessagePreview/MessagePreview.module.css:32]: Are these necessary?

File: src/app/components/MessagePreview/MessagePreview.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import UserImage from "../UserImage/UserImage";
+import styles from "./MessagePreview.module.css";
+
+function MessagePreview(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <UserImage imageSrc={"./User.jpeg"} className={styles.userImage} />
```
Comment (phlgr) [inline src/app/components/MessagePreview/MessagePreview.tsx:8]:
```suggestion
      <UserImage imageSrc="./User.jpeg" className={styles.userImage} />
```

Comment (infoucarsinan) [reply src/app/components/MessagePreview/MessagePreview.tsx:8]: done

File: src/app/components/MessagePreview/MessagePreview.module.css (hunk 1 above)
Comment (infoucarsinan) [reply src/app/components/MessagePreview/MessagePreview.module.css:4]: done

File: src/app/components/MessagePreview/MessagePreview.module.css (hunk 2 above)
Comment (infoucarsinan) [reply src/app/components/MessagePreview/MessagePreview.module.css:32]: done

**Pull request merged**